realm/realm-library/src/objectServer/java/io/realm/mongodb/mongo/MongoCollection.java

```diff
@@ -55,7 +65,7 @@
 public class MongoCollection<DocumentT> {
 
     private final MongoNamespace nameSpace;
-    private final OsMongoCollection<DocumentT> osMongoCollection;
+    final OsMongoCollection<DocumentT> osMongoCollection;
```

Comment:
Why remove private?

Reply:
Because the IDE suggested me to fix it and I blindly pressed yes.